app/src/main/java/com/bagi/flowkotlin/activity/covid19/DetailsCovidAdapter.kt

```diff
@@ -1,39 +1,33 @@
 package com.bagi.flowkotlin.activity.covid19
 
+import android.content.Context
 import android.view.LayoutInflater
 import android.view.ViewGroup
-import androidx.core.content.ContextCompat
-import androidx.databinding.DataBindingUtil
 import androidx.recyclerview.widget.RecyclerView
-import com.bagi.flowkotlin.R
 import com.bagi.flowkotlin.databinding.CardDetailsLayoutBinding
+import com.bagi.flowkotlin.model.Card
 
-class DetailsCovidAdapter (private val listDetails: MutableList<Triple<Int, Int?, Int?>>):
+class DetailsCovidAdapter(private val listDetails: MutableList<Card>) :
```

Comment:
Good job! but I wanted to demonstrate the Triple object of kotlin, I accept that.

Reply:
Thanks. i know that you had another intention and i of course accepts that. i had to change that in order to be able to use this object in data binding. "getString" method accepts only Int and variable you imported in the xml is Triple which is (Object, Object, Object). I can further elaborate on this if you ask.

Reply:
i see it's ok